activestorage/app/controllers/active_storage/blobs/proxy_controller.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 # Proxy files through application. This avoids having a redirect and makes files easier to cache.
+#
+# WARNING: All Active Storage controllers are publicly accessible by default. The
+# generated URLs are hard to guess, but permanent by design. If your files
+# require a higher level of protection consider implementing
+# [Authenticated Controllers](https://edgeguides.rubyonrails.org/active_storage_overview.html#authenticated-controllers).
```

Comment:
Is this how links work in RDoc? I have to double check. 🤔 

Anyways, once this is merged can you send a PR to update `s/edgeguides/guides` we can merge closer to a release -- there is no point in having broken links for the sake of not shipping a PR.

Reply:
Sorry, just fixed the links in RDoc based on https://github.com/rails/rails/blob/9053b8f77ffdb28f2494d773cec6fa1a84eceabe/activerecord/lib/active_record/relation.rb#L244
https://edgeapi.rubyonrails.org/classes/ActiveRecord/Relation.html

Reply:
https://github.com/rails/rails/pull/42599 is the follow up PR, I'll rebase it once this is merged